### PR TITLE
Bank transfer fee

### PR DESCRIPTION
**Pull request by wszarmach**

### CODE REVIEW

File: schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/payment/InvoicePaymentProvider.java [hunk 1]
```diff
@@ -15,12 +15,15 @@
...
 
+import java.math.BigDecimal;
+
 /**
  * @author krico
  * @since 05/08/15.
  */
 public class InvoicePaymentProvider implements PaymentProvider<InvoicePayment> {
     public static final int INVOICE_EXPIRES_DAYS = 3;
+    public static final double INVOICE_FEE_FLAT = new BigDecimal("1.00").doubleValue();
```
Comment (krico) [inline schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/payment/InvoicePaymentProvider.java]:
~~~java
BigDecimal.ONE.doubleValue();
//or even
1d;
~~~

File: schedule/schedule-appengine/src/main/resources/templates/partials/InvoicePaymentDetails.html.vm [hunk 2]
```diff
@@ -26,6 +26,12 @@
         </td>
     </tr>
     <tr>
+        <th #style("th") align="left">Fee:</th>
+        <td #style("td") align="left">
+            <nobr>$cut.formatCurrency($payment.Currency, $payment.Fee)</nobr>
```
Comment (krico) [inline schedule/schedule-appengine/src/main/resources/templates/partials/InvoicePaymentDetails.html.vm]: cool, you used `$cut`!!!

File: schedule/schedule-appengine/src/test/java/com/jasify/schedule/appengine/communication/CommunicatorTest.java [hunk 3]
```diff
@@ -139,6 +139,7 @@ public void testNotifyOfInvoiceCreated() throws Exception {
         InvoicePayment payment = new InvoicePayment();
         payment.setCurrency("CHF");
         payment.setAmount(200.12);
+        payment.setFee(1.2);
```
Comment (krico) [inline schedule/schedule-appengine/src/test/java/com/jasify/schedule/appengine/communication/CommunicatorTest.java]: We should test what happens with template if `fee == null`, or do we already?

**Comment by krico** [comment]
good PR!

File: schedule/schedule-appengine/src/test/java/com/jasify/schedule/appengine/communication/CommunicatorTest.java (hunk 3 above)
Comment (wszarmach) [inline schedule/schedule-appengine/src/test/java/com/jasify/schedule/appengine/communication/CommunicatorTest.java]: I did. Exception will occur. If we want null fee to work we will need some #if/#else/#end statements in the velocity template. At the moment I am expecting that fee is always set.

**Pull request merged**